src/views/user/ResetPassword/ForgotPassword.jsx

```diff
@@ -0,0 +1,153 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { PropTypes } from 'prop-types';
+import ForgotPasswordForm from './ForgotPasswordForm';
+import resetPasswordActions from '../../../actions/resetPasswordActions';
+import '../../../../public/styles/ResetPassword.scss';
+
+const { sendResetPasswordLink } = resetPasswordActions;
+
+/**
+ * The ForgotPassword component renders view to get
+ * user email and hits the route that sends reset
+ * password link to user email
+ * @class ForgotPassword
+ * @returns {undefined} this is a class thus does not return anything
+ */
+export class ForgotPassword extends React.Component {
+  state = {
+    email: '',
+    emailErrors: [],
+  };
+
+  /**
+   * Renders the component
+   * @param {object} email event object
+   * @returns {undefined} returns nothing
+   */
+  validate = (email) => {
```

Comment:
I had previously said something about not using arrow functions in jsx context, pls change this and bind to the constructor if neccessary